packages/backend-core/src/objectStore/objectStore.ts

```diff
@@ -146,12 +157,20 @@ export async function upload({
   path,
   type,
   metadata,
+  body,
+  addTTL,
 }: UploadParams) {
   const extension = filename.split(".").pop()
-  const fileBytes = fs.readFileSync(path)
+
+  const fileBytes = path ? (await fsp.open(path)).createReadStream() : body
```

Comment:
Like this here @samwho ?

Reply:
This looks perfect. You'll want to move from using `objectStore.upload` to `objectStore.streamUpload` further down as well 👍

Reply:
nvm sorted it